pages/index.js

```diff
@@ -1,15 +1,17 @@
 // file: pages/index.js
 
-import { useState, useEffect } from 'react';
+import { useState } from 'react';
 import Head from 'next/head';
 import Image from 'next/image';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
 import { motion } from 'framer-motion';
 import Book from '/components/Book';
+import { getCollection } from '../lib/mongodb';
```

Comment:
inconsistent import style:
- put the components imports together
- import from / don't use a relative ../

Reply:
oops looks like those were auto imported. fixed